docs/Node Operators/adding-jobs-to-your-node.md

```diff
@@ -1,67 +0,0 @@
----
```

Comment:
redirect

Reply:
does this exist somewhere else now?

Reply:
Nope but it is fully replaced by the [LINK Token Contracts](https://docs.chain.link/docs/link-token-contracts/).